compiler/esy.json

```diff
@@ -42,6 +43,7 @@
     "@opam/ounit": ">= 2.0.0"
   },
   "resolutions": {
+    "@grain/binaryen.ml": "link:../../binaryen.ml",
```

Comment:
Don't forget to use a git hash pointing at binaryen.ml so we can get the tests to pass here.